python/ray/util/accelerators/accelerators.py

```diff
@@ -3,5 +3,10 @@
 NVIDIA_TESLA_T4 = "T4"
 NVIDIA_TESLA_P4 = "P4"
 NVIDIA_TESLA_K80 = "K80"
-NVIDIA_TESLA_A100 = "A100"
 NVIDIA_TESLA_A10G = "A10G"
+
+NVIDIA_AMPERE_A100 = "A100"
```

Comment:
Sorry for missing this PR. Could you expose this symbol in `python/ray/util/accelerators/__init__.py`

Reply:
+1